.travis.yml

```diff
@@ -0,0 +1,22 @@
+---
+language: java
+script: ./gradlew clean :jObbifier:assemble
+
+os:
+    - linux
+
+jdk:
+    - openjdk7
+
+sudo: false
+deploy:
+  provider: releases
+  api_key: "${TRAVIS_TOKEN}"
```

Comment:
This basically is a configuration in Travis. Its most likely URL when you configure this would be:

https://travis-ci.org/monkey0506/jobbifier/settings

And you'd have to add an environment variable named TRAVIS_TOKEN there.

Github's side is probably here: https://github.com/monkey0506/jobbifier/settings/hooks - and you might even be able to configure a Token through there more easily than when I originally did.